ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,40 @@
-﻿namespace ConsoleApp1
+﻿using System.Collections;
+using static System.Net.Mime.MediaTypeNames;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Security.Principal;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
-		static void Main(string[] args)
+        static HttpClient client = new HttpClient();
+
+        static void Main(string[] args)
 		{
-		}
-	}
+            var result = GetShiba();
+
+            result.Wait();
```

Comment:
`Wait` will synchronously block until the task completes. So the current thread is literally blocked waiting for the task to complete.

Reply:
removed wait